### DISCUSSION TITLE
Scratch Blocks Blurred? Whyyyyy?

### DISCUSSION THREAD

**Discussion by nexensys** [ScratchAddons/ScratchAddons#3689] [Help] [answered]
I noticed this a while ago. I normally keep my blocks at a 15% opacity with the `transparent-orphans` addon, so I can see overlapping scripts. Recently I have noticed they had been blurred, all except for the one script on the top layer. I turned off the transparency and that seemed to fix it. Then, I tried running a script. As soon as the border showed up because it was running, the blocks became blurred. My first thought was that it was the addon that adds a more precise border around running blocks, So I disabled that too. Nope. Refreshed the page. No luck. Is this a Scratch block problem? (specifically asking, @apple502j and/or @GarboMuffin) Or do you think it is an addon?

![image](https://user-images.githubusercontent.com/70162741/138534445-3d770b84-694f-425b-9d34-41c80aef8ea3.png)
![image](https://user-images.githubusercontent.com/70162741/138534415-b6a25a42-f41d-40d9-9cf0-1b5581d9e256.png)

**Comment by nexensys** [comment, marked as answer]
sorry, should have been an issue.

**Comment by kccuber-scratch** [comment]
I believe this is a bug with some new Chrome update, as Bugs and Glitches on the Scratch forums has many of these topics.

**Comment by WorldLanguages** [comment]
This is due to a Chromium update
